src/General/Button/Button.test.tsx

```diff
@@ -15,6 +15,25 @@ it('<Button> should render with text "click me" and an onClick handler', () => {
   expect(ButtonSnapshot).toMatchSnapshot();
 });
 
+it('<Button> should render as button by default', () => {
+  const tag = 'a';
+  const { container } = render(<Button tag={tag}>test</Button>);
+  expect(container.querySelector(tag)).toBeTruthy();
+});
+
+describe('<Button> should render as the given tag', () => {
+  const tags = ['a', 'h1', 'div'];
+  tags.forEach((tag: string) => {
+    it(`<Button> should render as ${tag}`, () => {
+      const { container } = render(
+        <Button tag={tag as React.ElementType}>anchor button</Button>
```

Comment:
Opps I realied `anchor button` is misleading haha my bad. Can just be `button` instead